images/devtools-terraform-v1beta1/context/Makefile

```diff
@@ -117,7 +117,7 @@ validate-tf-$(1): | $(1)/.terraform $(if $(TF_NO_PROVIDERS),,$(1)/.terraform/pro
 	cd $(1) && $$(tfswitch)
 	cd $(1) && $$(terraform) fmt -check -recursive -diff
 	cd $(1) && $$(terraform) validate
-	test -e "$(1)/.tfsec-ignore" || test -e "$(1)/.trivy-ignore" || $$(trivy) $(1)
+	test -e "$(1)/.tfsec-ignore" || test -e "$(1)/.trivy-ignore" || $$(trivy) --ignorefile="$(1)/.trivyignore" "$(1)"
```

Comment:
this overlaps with the actually .trivyignore file. Can we have a different name? https://aquasecurity.github.io/trivy/v0.48/docs/configuration/filtering/#trivyignore